config/config.go

```diff
@@ -166,6 +167,7 @@ type (
 		ExternalHost   string   `yaml:"externalHost"`
 		ExternalPort   int      `yaml:"externalPort"`
 		BootstrapNodes []string `yaml:"bootstrapNodes"`
+		MasterKey      string   `yaml:"masterKey"`
```

Comment:
Instead of having another master key, can you feed libp2p with our pk

Reply:
Or do we worry that someone will run a fullnode w/o pk/sk?

Reply:
pls doc.